js/modules/WebAPIProvider/AuthAPI.js

```diff
@@ -25,6 +32,33 @@ define(function(require, exports) {
         return new Date(expirationInSeconds * 1000);
 
     });
+
+    const tokenExpired = ko.observable(false);
+    const askLoginOnTokenExpire = (function() {
```

Comment:
Token will be refreshed (when route is changed), but the tokenExpirationDate will remain the same, won't it?

Reply:
@johnSamilin , why should it if there is “subscribe” below?